<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainCalc</class>
 <widget class="QMainWindow" name="MainCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainCalc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="scream">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>611</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="general">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>301</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>General</string>
    </property>
    <widget class="QPushButton" name="equalButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="mathExpression">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="exponentialMembers">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>381</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Exponential Members</string>
    </property>
    <widget class="QLineEdit" name="yText">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="xText">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="yLabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="aText">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="bText">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="bLabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>b</string>
     </property>
    </widget>
    <widget class="QLabel" name="aLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>a</string>
     </property>
    </widget>
    <widget class="QLabel" name="xLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
    <widget class="QLabel" name="OperationLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Operation</string>
     </property>
    </widget>
    <widget class="QComboBox" name="operation">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Division</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Multiplication</string>
      </property>
     </item>
    </widget>
   </widget>
   <zorder>exponentialMembers</zorder>
   <zorder>scream</zorder>
   <zorder>general</zorder>
   <zorder>mathExpression</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
